apps/easy-messe/components/AboutUs/statisticMass.tsx

```diff
@@ -0,0 +1,39 @@
+import { theme } from "@easy-messe/libs/theme";
+import { Box, Typography } from "@mui/material";
+import { statisticMasses } from "./AboutUs";
+import { useIntl } from "react-intl";
+import { shortenNumber } from "@easy-messe/libs/utils"
+
+interface statisticProps {
+    statisticElement: statisticMasses
+}
```

Comment:
```suggestion
```

Statistic Element is the same as StatisticMasses.
just use StatisticMasses as the props interface.